.husky/pre-commit

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env sh
+. "$(dirname -- "$0")/_/husky.sh"
+
+npm run format
```

Comment:
use yarn

Reply:
looks like it does not support it https://yarnpkg.com/package/husky - recommendation is `npm`